_stbt/logging.py

```diff
@@ -158,12 +158,40 @@ def html(self, template, **kwargs):
                     .render(frame_number=self.frame_number)
                     .encode("utf-8"))
             f.write(jinja2.Template(dedent(template.lstrip("\n")))
-                    .render(**kwargs)
+                    .render(draw=self._draw, **kwargs)
                     .encode("utf-8"))
             f.write(jinja2.Template(_INDEX_HTML_FOOTER)
                     .render()
                     .encode("utf-8"))
 
+    def _draw(self, region, source_size, css_class, title=None):
+        import jinja2
+
+        if region is None:
+            return ""
+
+        if isinstance(css_class, bool):
+            if css_class:
+                css_class = "matched"
+            else:
+                css_class = "nomatch"
+
+        return jinja2.Template(dedent("""\
+            <div class="region {{css_class}}"
+                 style="left: {{region.x / image.width * 100}}%;
+                        top: {{region.y / image.height * 100}}%;
+                        width: {{region.width / image.width * 100}}%;
+                        height: {{region.height / image.height * 100}}%"
+                 {% if title %}
+                 title="{{ title | escape }}"
```

Comment:
Will this title thing do mouseover?

Reply:
Yes, but I removed it in a later refactoring.